tools/rapids-package-name

```diff
@@ -15,19 +15,26 @@ fi
 pkg_type="$1"
 
 append_cuda=0
-append_pyver=0
 append_wheelname=0
+append_pyver=0
+append_arch=0
```

Comment:
I'm surprised you didn't take this opportunity to alphabetize these :slightly_smiling_face:

Reply:
I'm sorting it by the order in which they appear in the filename.